src/version.h

```diff
@@ -19,7 +19,7 @@ static const int INIT_PROTO_VERSION = 209;
 static const int GETHEADERS_VERSION = 70077;
 
 //! disconnect from peers older than this proto version
-static const int MIN_PEER_PROTO_VERSION = PROTOCOL_VERSION;
+static const int MIN_PEER_PROTO_VERSION = 70006;
```

Comment:
Do we want this to be PROTOCOL_VERSION or at least 70007? We don't want to keep allowing 70006 wallets with improper signing.

Reply:
MIN_PEER_PROTO_VERSION gets set to 70006 to allow for an easier transition and ease syncing efforts during the update process.